client/src/partials/purchases/create/create.html

```diff
@@ -0,0 +1,207 @@
+                      {{ "FORM.LABELS.ON_PURCHASE" | translate }}
+                    </label>
+                  </div>
+                  <div class="radio">
+                    <label>
+                      <input type="radio" name="paymentMethod" id="on-delivery" value="FORM.LABELS.ON_DELIVERY" ng-model="PurchaseCtrl.order.details.payment_method" required>
+                      {{ "FORM.LABELS.ON_DELIVERY" | translate }}
+                    </label>
+                  </div>
+                  <div class="radio">
+                    <label>
+                      <input type="radio" name="paymentMethod" id="other" value="FORM.LABELS.OTHER" ng-model="PurchaseCtrl.order.details.payment_method" required>
+                      {{ "FORM.LABELS.OTHER" | translate }}
+                    </label>
+                  </div>
+
+                  <div class="help-block" ng-messages="PurchaseOrderForm.paymentMethod.$error" ng-show="PurchaseOrderForm.$submitted">
+                    <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                  </div>
+                </div>
+              </div>
+
+              <div class="col-md-6">
+
+                <bh-date-editor
+                  date-value="PurchaseCtrl.order.details.date"
+                  validation-trigger="PurchaseOrderForm.$submitted"
+                  min-date="PurchaseCtrl.minimumDate"
+                  max-date="PurchaseCtrl.timestamp">
+                </bh-date-editor>
+
+                <div
+                  class="form-group"
+                  ng-class="{'has-error' : PurchaseOrderForm.$submitted && PurchaseOrderForm.notes.$invalid }">
+                  <label class="control-label">{{ "FORM.LABELS.NOTES" | translate }}</label>
+                  <textarea
+                    ng-model="PurchaseCtrl.order.details.note"
+                    name="notes"
+                    class="form-control"
+                    rows="4"
+                    ng-maxlength="PurchaseCtrl.maxLength"
+                    required>
+                  </textarea>
+                  <div class="help-block" ng-messages="PurchaseOrderForm.notes.$error" ng-show="PurchaseOrderForm.$submitted">
+                    <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+
+
+    <!-- TODO Move padding to generic CSS class -->
+    <div class="row" style="padding-bottom : 5px;">
+      <div class="col-xs-12">
+
+        <!-- "Recover Items" button -->
+        <div class="btn-group pull-right" role="group">
+          <button
+            class="btn btn-default"
+            ng-class="{'btn-primary' : PurchaseCtrl.order.hasCacheAvailable() && PurchaseCtrl.order.details.supplier_uuid }"
```

Comment:
This could be a cool design pattern (changing class on availability) - it would be good to think about how we could make it uniform across the application! These kind of design decisions will have to be formalised soon before the first 2.x stable.